src/views/Permissions.vue

```diff
@@ -134,6 +134,7 @@ export default defineComponent({
       await this.store.dispatch('permission/getPermissionsByGroup', this.currentGroup.groupId)
       await this.store.dispatch('permission/checkAssociated')
       await this.getUsersCount()
+      await this.store.dispatch('permission/updateQuery', {queryString: '', showAllSelected: false})
```

Comment:
Handle the case when creating new security group.

Reply:
Added the code to clear query on creation of security group, Can verify on createSecurityGroup page.